MIP/mip-n/README.md

```diff
@@ -0,0 +1,148 @@
+# MIP-0: Release Conventions for Movement Technologies
+- **Description**: Release Conventions for Movement Technologies address the delivery of canonical artifacts to the Movement community.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-n](../MD/md-n)
+
+## Abstract
+
+Release Conventions for Movement Technologies standardize the process of delivering artifacts to the Movement community. By providing a coherent and repeatable process for releasing artifacts, these conventions ensure that Movement technologies are accessible and easy to use.
+
+The release conventions proposed herein are based on the usage of GitHub for source control and the tagging of GitHub releases. Guidelines for implementing programmatic access to release artifacts are also described. Finally, the intended relationship between releases and governance is outlined.
+
+## Motivation
+
+Movement technologies are continually evolving, and there's a need to ensure that the process of releasing artifacts is both organized and standardized. By establishing Release Conventions for Movement Technologies, we aim to facilitate the delivery of canonical artifacts to the Movement community. This will ensure that Movement technologies are accessible, easy to use, and well-vetted.
+
+## Specification
+
+The definitions of "artifact" and "release" are accepted from [MD-n](../MD/md-n) and submitted for formal adoption in [MG-n](../MG/mg-n) and [MG-k](../MG/mg-k). 
```

Comment:
The links in this sentence are broken. Not sure whether you meant to leave them as `md-n`, `mg-n`, and `mg-k` or not, I was thinking it would be more like a specific MD number.

Reply:
We can safely assign 23 now because the God of GitHub has granted this number to the PR.